astropy/io/fits/fitstime.py

```diff
@@ -259,13 +259,20 @@ def _verify_column_info(column_info, global_info):
     # considered.
     elif global_info['TREFPOS'] == 'TOPOCENTER':
 
-        column_info['location'] = global_info['location']
-        if column_info['location'] is None:
+        if global_info['location'] is None:
             warnings.warn(
```

Comment:
I actually don't see why this needs to generate a warning.  At least I think this simple case should work without warning:
```
In [7]: tm = Time(['2000:001'])
   ...: t = Table([tm])
   ...: t.write('time.fits', overwrite=True)
   ...: t2 = t.read('time.fits', astropy_native=True)
   ...: 
   ...: 
WARNING: Time column reference position "TRPOSn" is not specified. The default value for it is "TOPOCENTER", but due to unspecified observatory position, reference position will be ignored. [astropy.io.fits.fitstime]
```
I don't see any reason to assume that Time data written to FITS has anything to do with an observatory.

Reply:
We call is "space-time" because technically time alone does not have a meaning, e.g. if you see two signals in two bands (let's say you discover a supernova in I and R band), you might want to see which band detected it first. Due to the light-travel time, the position of the two observatories is required to do this for small time differences.
However, in practice, most observations are done somewhere on the earth and on a time precision much less than the light travel time, so location does not matter. Thus, I agree that a warning is needed in principle ,but I think it will be annoying for the vast majority of users where it does not matter, so I recommend "no warning".

Reply:
I think "no warning" makes sense. I was inclined to atleast warn the user following @hamogu's argument, but it definitely would be annoying.